### DISCUSSION TITLE
Actividad Docker - GRUPO 7

### DISCUSSION THREAD

**Discussion by bsantillan** [frlp-utn-ingsoft/catedra#110] [Show and tell]
https://github.com/IZelayeta/vetsoft

**Comment by rodriguezemautn** [comment]
Correcciones:
- el archivo Dockerfile se escribe con inicial mayuscula
- README.md no sugiere como se accede a la app
- el comando docker run sugerido con esta en detach (-d) al cerrar la consola se detiene el contenedor

Puntos aprobados:
*  construye la imagen
* buen tamaño de imagen
*despliega el contenedor (Pero no persiste)

Salida de comandos: 
$ docker ps  -a
2b2febc85899   vetsoft-app:Version   "python manage.py ru…"   2 minutes ago   Exited (0) About a minute ago             actividad-2
$ docker images -a
vetsoft-app   Version   1961b1fb97e6   3 minutes ago   310MB
